docs/Updating-npm-bundled-node-gyp.md

```diff
@@ -1,6 +1,6 @@
 # Updating the npm-bundled version of node-gyp
 
-**Note: These instructions are (only) tested and known to work with npm 8 and older.**
+**Note: These instructions are (only) tested and known to work with npm9 and older.**
```

Comment:
Do you have a reference for this? AFAIK npm is still on version 8.

Reply:
My mistake.  You are right!